kakao/src/main/kotlin/com/agoda/kakao/common/matchers/ListViewViewAdapterSizeMatcher.kt

```diff
@@ -0,0 +1,31 @@
+@file:Suppress("unused")
+
+package com.agoda.kakao.common.matchers
+
+import android.support.test.espresso.matcher.BoundedMatcher
+import android.view.View
+import android.widget.ListView
+import org.hamcrest.Description
+
+/**
+ * Matches ListView with count of children
+ *
+ * @param size of children count in ListView
+ */
+class ListViewViewAdapterSizeMatcher(private val size: Int) : BoundedMatcher<View, ListView>(ListView::class.java) {
```

Comment:
For some reason that I don't remember, the name of this class is wrong. It has double `View` in it. Could you please be so kind to change the name to `ListViewAdapterSizeMatcher`?